test/js/components/epics/detail.test.jsx

```diff
@@ -710,7 +710,11 @@ describe('<EpicDetail/>', () => {
           },
         },
       });
-      fireEvent.click(getByTitle('Remove'));
+      fireEvent.click(
+        getByTitle(
+          `Remove ${sampleGitHubUser2.name} (${sampleGitHubUser2.login})`,
```

Comment:
I don't see the stuff in parentheses in the title:

          `Remove ${sampleGitHubUser2.name} (${sampleGitHubUser2.login})`,

Why?